python_prototype/src/tqec/enums.py

```diff
@@ -0,0 +1,50 @@
+from enum import Enum, auto
+
+from tqec.position import Position
+
+
+class CornerPositionEnum(Enum):
```

Comment:
Why not use a network graph to represent the grid of qubits? That way, each qubit doesn't need to know its position of relative position to others -- it is encoded by the edges of the graph. There may be reasons for doing it this way, that are unclear to me though.

Reply:
The graph approach is more generic and may be simpler. On this specific enumeration, the "coordinates" are not really qubits, they are mostly multiplying factors that helps in computing the position of different corners in
https://github.com/QCHackers/tqec/blob/789ae72a22efa64904a661c7ab65769deea22d77/python_prototype/src/tqec/templates/orchestrator.py#L45-L48

Reply:
We can use this. You are right, using a graph library may compound the complexity.

I'd be curious to know how significant the coordinate system is to this project as it grows -- it may be worthwhile to support multiple, equivalent ones, but that may only need to be reflected on the frontend.